external/fv3fit/fv3fit/keras/_models/shared/callbacks.py

```diff
@@ -2,8 +2,21 @@
 import tensorflow as tf
 import wandb
 
+# Add third party callbacks here to make them available in training
+ADDITIONAL_CALLBACKS = {"WandbCallback": wandb.keras.WandbCallback}
 
-THIRD_PARTY_CALLBACKS = {"WandbCallback": wandb.keras.WandbCallback}
+
+def register_custom_callback(name: str):
+    """
+    Returns a decorator that will register the given training function
+    to be usable in training configuration.
```

Comment:
Decorators are generally confusing when describing their api usage in English, can you include an "examples" section in this docstring with a usage example?